protobuf/goss/goss.yaml

```diff
@@ -0,0 +1,3 @@
+command:
+  echo 1:
+    exit-status: 0
```

Comment:
@awekuit 
You should check that the `protobuf` command exists.

Reply:
@cw-ozaki 
OK ! 
I have Added some to `goss.yml`.
```sh
$ make test
docker-compose -f docker-compose.test.yml up --no-start sut
docker cp /Users/cw-akatsu/.ghq/src/github.com/chatwork/dockerfiles/protobuf/goss `basename $PWD`:/goss
docker-compose -f docker-compose.test.yml up --no-recreate sut
Starting protobuf ... done
Attaching to protobuf
protobuf   | INFO: Creating docker container
protobuf   | INFO: Copy goss files into container
protobuf   | INFO: Starting docker container
protobuf   | INFO: Container ID: 53de8e0b
protobuf   | INFO: Sleeping for 0.2
protobuf   | INFO: Container health
protobuf   | PID                 USER                TIME                COMMAND
protobuf   | 71205               root                0:00                tail -f /dev/null
protobuf   | INFO: Running Tests
protobuf   | File: /usr/bin/protoc: exists: matches expectation: [true]
protobuf   | File: /usr/bin/protoc: mode: matches expectation: ["0755"]
protobuf   | Command: /usr/bin/protoc --version: exit-status: matches expectation: [0]
protobuf   | Command: /usr/bin/protoc --version: stdout: matches expectation: [libprotoc 2.5.0]
protobuf   |
protobuf   |
protobuf   | Total Duration: 0.007s
protobuf   | Count: 4, Failed: 0, Skipped: 0
protobuf   | INFO: Deleting container
protobuf exited with code 0
```